koku/masu/processor/orchestrator.py

```diff
@@ -373,7 +374,20 @@ def start_manifest_processing(  # noqa: C901
                     queue=SUBS_EXTRACTION_QUEUE
                 )
                 LOG.info(log_json("start_manifest_processing", msg="created subs_task signature", schema=schema_name))
-                async_id = chord(report_tasks, group(summary_task, hcs_task, subs_task))()
+                ocp_on_cloud_trino_task = process_openshift_on_cloud_trino.s(
+                    provider_type=provider_type,
+                    schema_name=schema_name,
+                    provider_uuid=provider_uuid,
+                    tracing_id=tracing_id,
+                ).set(queue=SUMMARY_QUEUE)
```

Comment:
Just a question more than anything. We pass in the summary queue here but the default queue in the task is the download. Is that intentional?

Reply:
That is an excellent question and a good catch, any queue in particular you think this one belongs in? I'm not fully sure download makes a ton of sense for what is happening in this task but it's always been part of the download process so I'm not opposed to keeping it in download.

Reply:
Pushed a change for the default below to become the SummaryQueue since I agree with Cody on the reasoning.